docs/release.md

```diff
@@ -0,0 +1,22 @@
+# Status AddOn Relesea process 
+
```

Comment:
I think there should be some explanation somewhere that this repo is relying on goreleaser, with an explanation that goreleaser is triggered by new git tags, and maybe with pointers to the relevant configuration files.

Reply:
ok, done.